public/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial;
+  margin: 0 auto; /* Center website */
+  max-width: 800px; /* Max width */
+  padding: 20px;
+}
```

Comment:
Same here with overriding CSS since `body` is a top-level selector - consider using your service's ID as the selector and use `#reviews` instead.
The main site font used is Open Sans if you'd like to add that instead of Arial - [here's the link to the Google Fonts CDN.](https://fonts.google.com/specimen/Open+Sans) If you're using it because of a dependency, though, of course ignore this. Not sure if `chart.js` requires a certain font but wouldn't be surprised. You can grab the script tag directly out of my or Jay's service if that's easier as well.

Reply:
Good call here! I actually did away with chart.js, as it didn't end up being useful for my purposes. Anyway, I've updated the font and made #reviews the top-level selector. Thanks!